src/schema/v2/Alerts/index.ts

```diff
@@ -388,20 +388,6 @@ export const AlertType = new GraphQLObjectType<
       settings: {
         type: new GraphQLNonNull(AlertSettingsType),
       },
-
-      // Below fields are injected in the JSON when the alert data is being returned
-      // from ElasticSearch - currently when queried from the following places:
-      //
-      // - `alertsConnection` under `Artist`
-      // - `alertsSummaryArtistsConnection` under `Partner`.
-      totalUserSearchCriteriaCount: {
-        type: GraphQLInt,
-        resolve: ({ count_30d }) => count_30d,
-      },
-      hasRecentlyEnabledUserSearchCriteria: {
-        type: GraphQLBoolean,
-        resolve: ({ count_7d }) => count_7d && count_7d > 0,
-      },
```

Comment:
I think we can also remove the `count_30/7` that these guys are using as well

Ex: https://github.com/artsy/metaphysics/pull/6024/files#diff-87dc75f67d185f47ad8c6c3666c7cae7cd3f9ce3759d9145c386e58919ddef7eR166-R168

Reply:
and if we follow that back can probably remove these Edge types on Alert as well!

```
const AlertsEdgeFields = {
  counts: {
    type: new GraphQLObjectType({
      name: "AlertsCounts",
      fields: {
        totalUserSearchCriteriaCount: {
          type: GraphQLInt,
          resolve: ({ count_30d }) => count_30d,
        },
      },
    }),
    resolve: (x) => x,
  },
  isRecentlyEnabled: {
    type: GraphQLBoolean,
    resolve: ({ count_7d }) => count_7d > 0,
  },
}
```

Reply:
Update: we are pulling edge field retirement into separate ticket of work
https://artsyproduct.atlassian.net/browse/AMBER-1041